manifests/params.pp

```diff
@@ -22,5 +22,5 @@
   $user = 'qpidd'
   $group = 'qpidd'
 
-  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools']
+  $server_packages = ['qpid-cpp-server', 'qpid-cpp-client', 'python-qpid-qmf', 'python-qpid', 'policycoreutils-python', 'qpid-tools', 'qpid-cpp-server-linearstore']
```

Comment:
I know older qpids (which katello doesn't use anymore) use qpid-cpp-server-store instead of linear-store.  Do we want to try to support that?   (The answer doesn't have to be yes)